api/v1beta1/agentserviceconfig_types.go

```diff
@@ -109,12 +109,19 @@ type AgentServiceConfigSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="List of container registries without authentication"
 	// +optional
 	UnauthenticatedRegistries []string `json:"unauthenticatedRegistries,omitempty"`
+
 	// OSImageCACertRef is a reference to a config map containing a certificate authority certificate
 	// this is an optional certificate to allow a user to add a certificate authority for a HTTPS source of images
 	// this certificate will be used by the assisted-image-service when pulling OS images.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OS Image CA Cert ConfigMap reference"
 	// +optional
 	OSImageCACertRef *corev1.LocalObjectReference `json:"OSImageCACertRef,omitempty"`
+
+	// OSImagesAdditionalParamsRef is a reference to a secret containing a headers and query parameters to be used during OS image fetch.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OS Images additional parameters reference"
+	// +optional
+	OSImagesAdditionalParamsRef *corev1.LocalObjectReference `json:"OSImagesAdditionalParamsRef,omitempty"`
```

Comment:
Sorry for the late change, but this should probably be consistent with the previous name.

So `OSImageAdditionalParamsRef`